Simplenote/SPTracker+Extensions.swift

```diff
@@ -27,6 +27,58 @@ extension SPTracker {
     static func trackVerificationDismissed() {
         trackAutomatticEvent(withName: "verification_dismissed", properties: nil)
     }
+
+    static func trackNotice(_ value: String) {
+        trackAutomatticEvent(withName: "notice_presented", properties: ["notice": value])
+    }
+
+    static func trackNoticeAction(_ value: String, action: String) {
+        trackAutomatticEvent(withName: "notice_action_tapped", properties: ["notice": value, "notice_action": action])
+    }
+
+    static func trackNoticeLinkCopied() {
+        trackNotice("Link Copied")
```

Comment:
For consistency reasons (with the naming) perhaps:

- All of the `notice names` should be lowecase
- Preferrably a single word, if multiple, maybe using `_` as a word spacer